crates/pathfinder/src/cairo/ext_py.rs

```diff
@@ -0,0 +1,368 @@
+//! External python process pool for execute calls.
+//!
+//! The python processes are executing `$REPO_ROOT/py/src/call.py` and communicate over by sending
+//! and receiving json + `'\n'`. Main entry point is the [`service::start`] which manages running
+//! given number of N processes. The python script uses sqlite to read pathfinder's database, which
+//! should not cause issues in WAL mode.
```

Comment:
So does this mean we are enabling WAL?

Reply:
I'm still in favor of us first witnessing this freeze up the sync process, then maybe raise the sqlite related timeout (can't remember right now which was it), and only then enable the WAL. Maybe I'll do a tracing round up updates around the same time, but at the moment we can't test this.

Reply:
Resolving for unchanged stance.